portal/index.php

```diff
@@ -433,23 +441,12 @@ function restoreSession(){
             });
             <?php } ?>
             <?php if (isset($_GET['w'])) { ?>
-            var unique_id = $.gritter.add({
-                title: '<span class="red">' + <?php echo xlj('Oops!');?> +'</span>',
-                text: <?php echo xlj('Something went wrong. Please try again.'); ?>,
-                sticky: false,
-                time: '5000',
-                class_name: 'my-nonsticky-class'
-            });
+                // mdsupport - Would be good to include some clue about what went wrong!
```

Comment:
From a security perspective, this message should probably be generic no matter the login issue (could make it more helpful by providing a contact option when unable to login though).

Reply:
Also not sure if telling them to keep trying to login makes sense. Maybe the message should simply state that login was not successful (along with contact option if that is set)